### PR TITLE
Add Backup/Usage Page

### PR DESCRIPTION
**Pull request by gabedealmeida**

### CODE REVIEW

**Comment by super3** [comment]
@gabedealmeida Looks like both buttons are selected when you click backup.

Capture

**Review by super3 — APPROVED** [review]
Tested any everything looks good here.

File: ui/src/store/account.js [hunk 1]
```diff
@@ -25,7 +27,14 @@ export default {
 			apiKey: null,
 			projectId: null,
 			errorMessage: null,
-			loggingIn: false
+			loggingIn: false,
+			endpoint: "https://us1.storj.io",
+			accessKey: "4EdkanJHIlJ8949JIk23jlfjaJNallfH90",
```
Comment (montyanderson) [inline ui/src/store/account.js:32]: What are these keys?

**Review by montyanderson — CHANGES_REQUESTED** [review]

File: ui/src/store/account.js (hunk 1 above)
Comment (super3) [reply ui/src/store/account.js:32]: Those are dummy keys for now.

**Review by super3 — CHANGES_REQUESTED** [review]
Need to make the backup page functional. It is ok if the usage page is not functional at this time.

**Review by montyanderson — APPROVED** [review]

**Review by super3 — APPROVED** [review]

**Pull request merged**